### PR TITLE
Code review request

### PR DESCRIPTION
**Pull request by JordanLikesLego**
check formatting, naming and logic

@JackMcKill

### CODE REVIEW

File: Prac_05/commit4jack.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+words = string.split()
+
+longest = len(max(words, key=len))
+
+for word in words:
+    if word in counts:
+        counts[word] += 1
+    else:
+        counts[word] = 1
+
```
Comment (JackMcGill) [inline Prac_05/commit4jack.py:12]: There is an extra blank line here when there should only be one

Comment (JordanLikesLego) [reply Prac_05/commit4jack.py:12]: ok i will fix

Comment (JackMcGill) [reply Prac_05/commit4jack.py:12]: Thanks

**Review by JordanLikesLego — COMMENTED** [review]
all done

**Review by JackMcGill — APPROVED** [review]
well done

**Pull request closed without merging**